queries/models/financeiro/subsidio_faixa_servico_dia_tipo_viagem.sql

```diff
@@ -0,0 +1,158 @@
+    AND DATE("{{ var("end_date") }}")
+  ),
+  indicador_ar AS (
+  SELECT
+    data,
+    id_veiculo,
+    status,
+    SAFE_CAST(JSON_VALUE(indicadores,"$.indicador_ar_condicionado") AS BOOL) AS indicador_ar_condicionado
+  FROM
+    -- {{ ref("sppo_veiculo_dia") }}
+    rj-smtr.veiculo.sppo_veiculo_dia
+  WHERE
+    data BETWEEN DATE("{{ var("start_date") }}")
+    AND DATE("{{ var("end_date") }}")
+  ),
+  viagem AS (
+  SELECT
+    data,
+    servico_realizado AS servico,
+    id_veiculo,
+    id_viagem,
+    datetime_partida
+  FROM
+    -- {{ ref("viagem_completa") }}
+    rj-smtr.projeto_subsidio_sppo.viagem_completa
+  WHERE
+    data BETWEEN DATE("{{ var("start_date") }}")
+    AND DATE("{{ var("end_date") }}")
+  ),
+  ar_viagem AS (
+  SELECT
+    v.data,
+    v.servico,
+    v.id_viagem,
+    v.datetime_partida,
+    COALESCE(ia.indicador_ar_condicionado, FALSE) AS indicador_ar_condicionado
+  FROM
+    viagem v
+  LEFT JOIN
+    indicador_ar ia
+  ON
+    ia.data = v.data
+    AND ia.id_veiculo = v.id_veiculo
+  ),
+  subsidio_servico_ar AS (
+    SELECT
+      sfd.data,
+      sfd.tipo_dia,
+      sfd.faixa_horaria_inicio,
+      sfd.faixa_horaria_fim,
+      sfd.consorcio,
+      sfd.servico,
+      sfd.pof,
+      COALESCE(s.tipo_viagem, "Sem viagem apurada") AS tipo_viagem,
+      s.id_viagem,
+      s.distancia_planejada,
+      s.subsidio_km,
+      s.subsidio_km_teto,
+      s.indicador_viagem_remunerada,
+      CASE
+        WHEN s.tipo_viagem = "Autuado por ar inoperante" THEN TRUE
+        WHEN s.tipo_viagem = "Licenciado sem ar e não autuado" THEN TRUE
+        WHEN sfd.pof < 60 THEN TRUE
+        WHEN s.tipo_viagem = "Registrado com ar inoperante" THEN TRUE
+        ELSE FALSE
+      END AS indicador_penalidade_judicial,
+      COALESCE(av.indicador_ar_condicionado, FALSE) AS indicador_ar_condicionado
+    FROM
+      subsidio_faixa_dia AS sfd
+    LEFT JOIN
+      ar_viagem AS av
+    ON
+      sfd.data = av.data
+      AND sfd.servico = av.servico
+      AND av.datetime_partida BETWEEN sfd.faixa_horaria_inicio
+      AND sfd.faixa_horaria_fim
+    LEFT JOIN
+      servico_km_apuracao AS s
+    ON
+      sfd.data = s.data
+      AND sfd.servico = s.servico
+      AND s.id_viagem = av.id_viagem
+  )
+SELECT
+  data,
+  tipo_dia,
+  faixa_horaria_inicio,
+  faixa_horaria_fim,
+  consorcio,
+  servico,
+  indicador_ar_condicionado,
+  tipo_viagem,
+  SAFE_CAST(COALESCE(COUNT(id_viagem), 0) AS INT64) AS viagens_faixa,
+  SAFE_CAST(COALESCE(SUM(distancia_planejada), 0) AS NUMERIC) AS km_apurada_faixa,
+  SAFE_CAST(COALESCE(SUM(IF(tipo_viagem != "Não licenciado", distancia_planejada, 0)), 0) AS NUMERIC) AS km_subsidiada_faixa,
+  SAFE_CAST(SUM(IF(indicador_viagem_remunerada = TRUE AND pof >= 80, distancia_planejada*subsidio_km, 0)) AS NUMERIC) AS valor_apurado,
+  SAFE_CAST(COALESCE(SUM(IF(indicador_viagem_remunerada = TRUE, 0, distancia_planejada*subsidio_km)), 0) AS NUMERIC) AS valor_acima_limite,
```

Comment:
Faz sentido essa lógica? A viagem acima do limite não seria `indicador_viagem_remunerada = FALSE AND tipo_viagem != "Sem transação"`? Acredito que faz mais sentido criar um indicador na CTE anterior sobre isso